pool/dicePool.go

```diff
@@ -0,0 +1,32 @@
+package pool
+
+import (
+	"time"
+	"github.com/panjf2000/ants/v2"
+)
+// Type alias for the ant pools
+type DicePool = ants.PoolWithFunc
+
+func init() {
+	ants.Release()
+}
+
+const (
+	// Max number of workers 256 * 1024
+	DefaultPoolSize = 1 << 18
+
+	// If we should wait when pool workers ain't available
+	// false make it won't wait
+	Nonblocking = false
+
+	// ExpiryDuration is the interval time to clean up those expired workers.
+	ExpiryDuration = 10 * time.Second
```

Comment:
DefaultPoolSize -> MaxConcurrency and let the default value be part of the config.
Expiry duration can also be part of the config. Nonblocking should always be true. Don't see a reason for it to be configurable.